spec/std/isa/csr/mstatus.yaml

```
@@ -65,6 +65,26 @@ fields:
     definedBy: Smdbltrp
     type: RW-H
     reset_value: UNDEFINED_LEGAL
+  SDT:
+    location: 41
+    base: 64
+    description: |
+      *Supervisor Disable Trap*
```

We missed a "long_name" conversion, but I think the name should be different as well:
```suggestion
    long_name: Supervisor Double Trap
    description: |
```